Form/FormFlow.php

```diff
@@ -156,6 +157,18 @@
 	private $genericFormOptions = array();
 
 	/**
+	 * Form was determined to be stale.
+	 * @var boolean
+	 */
+	private $stale = false;
```

Comment:
Maybe we can find a better word than "stale", something more self-explaining. You already use "expired" in the error message. How about that?